submissions/nadsatt/friends-app/modules/image-service.js

```diff
@@ -0,0 +1,93 @@
+export class ImageService {
```

Comment:
Logic of downloading of icons should be in `APIService` and `ImageService` should call `fetchImages` and then handle them in its own specific way.

`APIService` - fetches data.
`ImageService` - works with results.